### PR TITLE
hydra: added rewrites for subpath extension/

### PR DESCRIPTION
**Pull request by tpluscode**
Draft until we're ready with the redirect targets (hydracg/extensions#7)

### CODE REVIEW

**Comment by tpluscode** [comment]
@alien-mcl please have a look

@lanthaler please let us know what is your intention for https://w3id.org/hydra

**Review by alien-mcl — APPROVED** [review]

File: hydra/extension/.htaccess
Comment (davidlehn) [inline hydra/extension/.htaccess:24]: Might need to escape that `+`?

**Comment by davidlehn** [comment]
The `hydra` endpoint is one of the earliest and could use a quick update.  While this dir is being worked on, please also add maintainers in comments to `hydra/.htaccess`, the `.htaccess` here, and/or a readme.

File: hydra/extension/.htaccess
Comment (tpluscode) [reply hydra/extension/.htaccess:24]: Indeed, thanks.

File: hydra/README.md
Comment (davidlehn) [inline hydra/README.md:10]: It would better to list the maintainers.  In this case I'm guessing it's the same, but better to be explicit.  Also would help to list github user ids.

**Comment by dgarijo** [comment]
@tpluscode,
we would need the approval from the original creator @lanthaler to go forward with these changes.

**Comment by dgarijo** [comment]
@tpluscode 
any news on this pr?

**Comment by tpluscode** [comment]
hey, thanks for reminding me. I sent Markus an email. I think he filters out GH notifications

File: hydra/extension/.htaccess
Comment (lanthaler) [inline hydra/extension/.htaccess:21]: Nit: serilizations -> serializations & documen -> documents

**Comment by lanthaler** [comment]
Thanks for pinging me over mail @tpluscode. The changes look good to me

**Pull request merged**
